pallets/dispenser/src/benchmarking.rs

```diff
@@ -33,6 +33,7 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::RuntimeEvent) {
 #[benchmarks]
 mod benchmarks {
 	use super::*;
+	use polimec_common_test_utils::get_mock_jwt_with_cid;
```

Comment:
Think this breaks the dispenser benchmarks as code is still using get_mock_jwt or it is an unnecessary import?

Reply:
It was an unnecessary import https://github.com/Polimec/polimec-node/pull/258/commits/fcac9f47fab3a9d0941382729793227a17e6da16